torch/_dynamo/eval_frame.py

```diff
@@ -269,12 +269,16 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         self.dynamic_ctx.__exit__(exc_type, exc_val, exc_tb)
         self.backend_ctx.__exit__(exc_type, exc_val, exc_tb)
 
-    def __call__(self, fn):
+    def __call__(self, fn, fx_node_meta_context=None):
         # public api for compiler config/options
         def get_compiler_config():
             return self.compiler_config
 
         fn = innermost_fn(fn)
+
+        if isinstance(fn, torch.fx.GraphModule):
+            fn.forward.__func__._torchdynamo_context = {"orig_graphmodule": fn}
```

Comment:
If we inline into a callable corresponding to a GraphModule we should probably still be able to reestablish this context. This makes the tracking robust to changes on the enclosing code

Reply:
done